static/js/impala/stats/manager.js

```diff
@@ -518,9 +520,14 @@ z.StatsManager = (function() {
             key = parts[0];
         parts = parts.slice(1);
 
+        // Fenix app versions are build IDs so we need a prefix for end users.
+        var versionPrefix = metric === 'apps' &&
+            key === '{aa3c5121-dab2-40e2-81ca-7ea25febc110}' &&
+            useFenixBuildIDs ? 'Build #' : '';
+
         if (metric in csv_keys) {
             if (key in csv_keys[metric]) {
-                return csv_keys[metric][key] + ' ' + parts.join(' ');
+                return csv_keys[metric][key] + ' ' + versionPrefix + parts.join(' ');
```

Comment:
what's this actually doing?  The end result in the UI?

Reply:
sorry, added a screenshot now